src/main.ts

```diff
@@ -153,7 +153,7 @@ export function generateFile(typeMap: TypeMap, fileDesc: FileDescriptorProto, pa
     },
     options
   );
-  if (hasAnyTimestamps) {
+  if (hasAnyTimestamps && options.outputJsonMethods) {
```

Comment:
Is this correct? I see `toTimestamp` and `fromTimestamp` being used in the `generateEncode` and `generateDecode` functions.

Reply:
You are correct. I need to be more granular.
```suggestion
  if (hasAnyTimestamps && (options.outputJsonMethods || options.outputEncodeMethods) {
```
then in `addTimestampMethods(file: FileSpec, options: Options): FileSpec` add the `fromJsonTimestamp` conditionally.